src/mito2/src/memtable/merge_tree/tree.rs

```diff
@@ -335,10 +345,18 @@ impl Drop for TreeIter {
         MERGE_TREE_READ_STAGE_ELAPSED
             .with_label_values(&["fetch_next_partition"])
             .observe(self.metrics.fetch_partition_elapsed.as_secs_f64());
+        let scan_elapsed = self.metrics.iter_elapsed.as_secs_f64();
+        READ_STAGE_ELAPSED
+            .with_label_values(&["scan_memtable"])
+            .observe(scan_elapsed);
         log::debug!(
```

Comment:
We should use `common_telemetry::debug` log the `debug` macro under `common_telemetry::logging`.

Reply:
fixed https://github.com/GreptimeTeam/greptimedb/pull/3404/commits/755736d46ad4ee3a4091aaa2b592c9f8dcc1a302#diff-c49c8c96fe96144915d2d855220956e3284752035404a1a9b230934e2ec213deR352